react-backend/router.js

```diff
@@ -40,7 +40,8 @@ router.use('/cb/pwd/change', cbPasswordChange);
 router.use('/cb/pwd/reset', cbPasswordReset);
 
 // Not open but following api plan
-router.post('/cb/feedback', mwIsAuthenticated, cbFeedback);
+router.get('/cb/feedback', mwAdminIsAuthenticated, cbFeedback.get);
```

Comment:
I would take this opportunity to start removing things from this router file. My intention was for the `controllers` directory in each "feature" directory to export a router object.

So that means `/cb-admin/controllers` exports a router containing all `/api/cb` routes.

Reply:
Can we move all files in [one big swoop?](https://github.com/TwinePlatform/DataPower/issues/296) I'm starting to have some sad rebasing woes.

Reply:
These are new routes, they shouldn't cause any rebase issues. What problems are you having?

Reply:
I just keep loosing bit's n pieces 😕.